example/ios/Podfile

```diff
@@ -7,7 +7,7 @@ platform :ios, '8.0'
 target 'AirMapsExplorer' do
   rn_path = '../../node_modules/react-native'
 
-  pod 'yoga', path: "#{rn_path}/ReactCommon/yoga/yoga.podspec"
+  pod 'Yoga', path: "#{rn_path}/ReactCommon/yoga/yoga.podspec"
```

Comment:
This was recently changed to lower case.
https://github.com/react-community/react-native-maps/pull/1816

Reply:
it won't build with uppercase

Reply:
I'm guessing this is because the example runs on an older version of react native.
Would be great to bump it up to the latest but will likely require a bit more work.

Reply:
I'll try 😊